backend/routers/field_officer.py

```
@@ -484,54 +533,61 @@ def get_visit_statistics(db: Session = Depends(get_db)):
 def verify_visit(
     visit_id: int,
     verifier_email: str = Form(..., description="Email of verifying admin/supervisor"),
-    db: Session = Depends(get_db)
+    db: Session = Depends(get_db),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)